utils/init_functions_config.py

```diff
@@ -0,0 +1,155 @@
+# -- Base Python imports
+import os
+import logging
+
+
+# -- Functions
+def check_if_config_exists(project_root: str, config_name: str = "testing_config.yml"):
+    """Function to check if a configuration file already exists and ensure that the user
+     wants it to be overwritten, if that is the case"""
+    if config_name in os.listdir(project_root):
+        logging.info(
+            f"A Grater Expectations configuration file ({config_name}) already exists "
+            "in this directory. Are you sure you want to initialize a new file and "
+            "overwrite the existing one (y/[n])?"
+        )
+        response = input("Input: ")
+        if response in ["y", "Y", "yes", "Yes", "YES"]:
```

Comment:
I would do:
`if response.lower() in ["y", "yes"]:`
to cater for different combinations of yes e.g YeS

Reply:
Done